fabric-api-base/src/main/java/net/fabricmc/fabric/api/event/EventFactory.java

```diff
@@ -138,4 +119,24 @@ public static <T> Event<T> createWithPhases(Class<? super T> type, Function<T[],
 	public static String getHandlerName(Object handler) {
 		return handler.getClass().getName();
 	}
+
+	/**
+	 * @deprecated Always returns {@code false}, do not use.
+	 */
+	@Deprecated(forRemoval = true)
```

Comment:
I dont think we should set `forRemoval`

Reply:
I made it `forRemoval` to give a strong warning (especially since `invalidate` should be removed, imo), but maybe this one can stay forever (it's a trivial method anyway).